prow/integ-suite-kind.sh

```diff
@@ -38,6 +38,8 @@ source "${ROOT}/common/scripts/kind_provisioner.sh"
 
 TOPOLOGY=SINGLE_CLUSTER
 NODE_IMAGE="gcr.io/istio-testing/kind-node:v1.23.0"
+[[ "$(uname -m)" == "aarch64" ]] && NODE_IMAGE="kindest/node:v1.23.0"
```

Comment:
We should just build ours as multi-arch. will push one

Reply:
Will be gcr.io/istio-testing/kind-node:v1.23.4, working on building it

Reply:
Thanks

Reply:
Can you update to 1.23.4

Reply:
OK

Reply:
Updated.